### PR TITLE
DarrenS: Batch appointments & microfixes

### PR DESCRIPTION
**Pull request by dstults**

### CODE REVIEW

File: src/main/java/edu/lwtech/csd297/teachersfirst/actions/DeleteOpeningAction.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public void RunAction() {
 		//logger.info(DataManager.getOpeningDAO().size() + " records total");
 		logger.debug("Deleted opening ID: [{}]", openingIdInt);
 		
-		this.SendPostReply("/openings", "", "Opening %5B" + openingIdString + "%5D deleted!");
+		this.SendPostReply("/openings", "", "Opening %5B" + openingIdString + "%5D, deleted!");
```
Comment (DranKof) [inline src/main/java/edu/lwtech/csd297/teachersfirst/actions/DeleteOpeningAction.java:46]: This seems... from out of nowhere?

File: src/main/java/edu/lwtech/csd297/teachersfirst/pages/MakeAppointmentPage.java [hunk 2]
```diff
@@ -44,12 +44,11 @@ public void loadPage() {
 			final String openingEndTimeString = QueryHelpers.getGet(request, "openingEndTime");
 			final String appointmentStartTimeString = QueryHelpers.getGet(request, "appointmentStartTime");
 			final String appointmentEndTimeString = QueryHelpers.getGet(request, "appointmentEndTime");
-			//final String appointmentDuration = QueryHelpers.getGet(request, "appointmentDuration");
 			
 			// If thrown here on a "go back", assign these:
 			String defaultStartTime = "";
 			String defaultDuration = "";
-			if (appointmentStartTimeString != "" && appointmentEndTimeString != "") {
+			if (!appointmentStartTimeString.equals("") && !appointmentEndTimeString.equals("")) {
```
Comment (DranKof) [inline src/main/java/edu/lwtech/csd297/teachersfirst/pages/MakeAppointmentPage.java:51]: Nice catch. Java be fussy.

File: src/main/java/edu/lwtech/csd297/teachersfirst/pages/PageLoader.java [hunk 3]
```diff
@@ -85,6 +85,7 @@ protected PageLoader(HttpServletRequest request, HttpServletResponse response) {
 		templateDataMap.put("isAdmin", isAdmin);
 		templateDataMap.put("isInstructor", isInstructor);
 		templateDataMap.put("isStudent", isStudent);
+		message = message.replace("//", "<br>\n			");
```
Comment (DranKof) [inline src/main/java/edu/lwtech/csd297/teachersfirst/pages/PageLoader.java:88]: The tab count there was perfect, was that just lucky?

File: src/main/java/edu/lwtech/csd297/teachersfirst/pojos/Appointment.java [hunk 4]
```diff
@@ -15,17 +15,32 @@
...
+		this(-1, studentID, instructorID, DateHelpers.toTimestamp(year, month, startDay, startHour, startMinute, 0),
+				DateHelpers.toTimestamp(year, month, endDay, endHour, endMinute, 0));
 	}
 	
 	public Appointment(int studentID, int instructorID, Timestamp startTime, Timestamp endTime) {
 
 		this(-1, studentID, instructorID, startTime, endTime);
 	}
+	
+	public Appointment(PlannedAppointment plan) {
```
Comment (DranKof) [inline src/main/java/edu/lwtech/csd297/teachersfirst/pojos/Appointment.java:31]: Sexy.

**Review by DranKof — APPROVED** [review]

File: src/main/java/edu/lwtech/csd297/teachersfirst/actions/DeleteOpeningAction.java (hunk 1 above)
Comment (dstults) [reply src/main/java/edu/lwtech/csd297/teachersfirst/actions/DeleteOpeningAction.java:46]: Yeah, I don't want to make a commit just to undo that, not sure why it got put in though.

File: src/main/java/edu/lwtech/csd297/teachersfirst/pages/PageLoader.java (hunk 3 above)
Comment (dstults) [reply src/main/java/edu/lwtech/csd297/teachersfirst/pages/PageLoader.java:88]: Yes!

**Pull request merged**